python/cudf/cudf/core/column/string.py

```diff
@@ -4693,36 +4625,9 @@ def _nbytes(self):
             return self.children[1].size
 
     def as_numerical_column(self, dtype, **kwargs):
```

Comment:
Remove `kwargs` here?

Reply:
Good catch - tried doing that and running tests right now.